tests/integration_tests/test_ray.py

```diff
@@ -353,62 +339,45 @@ def test_ray_save_outputs(dataset_type, ray_cluster_2cpu):
 
 
 @pytest.mark.distributed
-@pytest.mark.parametrize(
-    "df_engine",
-    [
-        "dask",
-        pytest.param(
-            "modin",
-            marks=pytest.mark.skipif(_modin_ray_incompatible, reason="modin<=0.15.2 does not support ray>=1.13.0"),
-        ),
-    ],
-)
-def test_ray_tabular(df_engine, ray_cluster_2cpu):
+@pytest.mark.parametrize("dataset_type", ["csv", "parquet"])
+def test_ray_text_sequence_timeseries(ray_cluster_2cpu, dataset_type):
     input_features = [
+        text_feature(),
         sequence_feature(encoder={"reduce_output": "sum"}),
-        category_feature(encoder={"vocab_size": 2}, reduce_input="sum"),
-        number_feature(normalization="zscore"),
-        set_feature(),
-        binary_feature(),
-        bag_feature(),
-        vector_feature(),
-        h3_feature(),
-        date_feature(),
+        timeseries_feature(),
     ]
     output_features = [
-        binary_feature(bool2str=["No", "Yes"]),
         binary_feature(),
-        number_feature(normalization="zscore"),
+        # text_feature(decoder={"vocab_size": 3}),  # Error having to do with a missing key: MLI-70
```

Comment:
I would open an issue in GH and remove references to internal issue trackers.

Reply:
Done.